src/offlinelogstorage/dlt_offline_logstorage.c

```diff
@@ -1195,6 +1207,11 @@ DLT_STATIC DltLogstorageFilterConf
         .key = "SpecificSize",
         .func = dlt_logstorage_check_specificsize,
         .is_opt = 1
+    },
+    [DLT_LOGSTORAGE_FILTER_CONF_GZIP_COMPRESSION] = {
```

Comment:
You should add all new options into the example configuration, so they are documented

Reply:
Added them to `doc/dlt_offline_logstorage.md`